examples/stable/mysql-csi-snapshot/README.md

```diff
@@ -0,0 +1,267 @@
+- CSI Driver installed
+- VolumeSnapshotClass and StorageClass resources defined (uses above CSI Driver)
+- Snapshot APIs enabled
+
+## Install MySQL
+
+Install the MySQL database using the bitnami chart with the release name `mysql-release`.
+
+```bash
+# Add bitnami in your local chart repository
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+
+# Update your local chart repository
+$ helm repo update
+
+# Install the MySQL database (Helm Version 3)
+$ kubectl create namespace mysql
+$ helm install mysql-release bitnami/mysql --namespace mysql \
+    --set auth.rootPassword='asd#45@mysqlEXAMPLE' \
+    --set architecture="standalone"
+```
+
+Above command deploys a MySQL instance in the `mysql` namespace.
+
+To retrieve your root password run the following command.
+
+```bash
+$ kubectl get secret mysql-release --namespace mysql -o jsonpath="{.data.mysql-root-password}" | base64 --decode`
+```
+
+> **Tip**: List all releases using `helm list --all-namespaces`, using Helm Version 3.
+
+# Create Application data
+
+Connect to the MySQL database.
+
+```bash
+# Run shell inside MySQL's pod
+$ kubectl exec -it $(kubectl get pods -n mysql --selector=app.kubernetes.io/instance=mysql-release -o=jsonpath='{.items[0].metadata.name}') -n mysql -- bash
+```
+
+From inside the shell, use the mysql CLI to insert some data into the test database.
+
+```bash
+$ mysql --user=root --password=asd#45@mysqlEXAMPLE
+
+mysql> CREATE DATABASE test;
+Query OK, 1 row affected (0.00 sec)
+
+mysql> USE test;
+Database changed
+
+# Create "employees" table
+mysql> CREATE TABLE employees (name VARCHAR(20), manager VARCHAR(20), address VARCHAR(200), sex CHAR(1), birth DATE, department VARCHAR(20));
+Query OK, 0 rows affected (0.02 sec)
+
+# Insert row to the table
+mysql> INSERT INTO employees VALUES ('Max','Tom','Naperville, Illinois, Chicago','m','1999-03-30','Engineering');
+Query OK, 1 row affected (0.01 sec)
+
+# View data in "employees" table
+mysql> SELECT * FROM employees;
++------+---------+-------------------------------+------+------------+-------------+
+| name | manager | address                       | sex  | birth      | department  |
++------+---------+-------------------------------+------+------------+-------------+
+| Max  | Tom     | Naperville, Illinois, Chicago | m    | 1999-03-30 | Engineering |
++------+---------+-------------------------------+------+------------+-------------+
+1 row in set (0.00 sec)
+```
+
+# Backup MySQL DB
+
+## Create Blueprint
+
+Create Blueprint in the same namespace as the Kanister controller.
+
+> **Note**: We used a Kubernetes cluster on DigitalOcean. Hence, snapshotClass and storageClass in the ./mysql-csi-snapshot-bp.yaml file is set to `do-block-storage`. Please correct these arguments as per your cluster setup. You can make use of `kubectl patch` or `kubectl edit` commands to make these changes.
+
+```bash
+$ kubectl create -f ./mysql-csi-snapshot-bp.yaml -n kanister
```

Comment:
file is named mysql-csi-snapshot-**blueprint**.yaml

Reply:
Oh. My bad. I missed that.